src/parse-stream.js

```diff
@@ -105,7 +105,8 @@ export default class ParseStream extends Stream {
     if (line.indexOf('#EXT') !== 0) {
       this.trigger('data', {
         type: 'comment',
-        text: line.slice(1)
+        text: line.slice(1),
+        line
```

Comment:
We wanted to add the entire line hashtag and all to the tags array for each tag and comment.  Even though comments were being ignored previously they still had the text property so I thought to keep it in. I can remove the text property.